### DISCUSSION TITLE
Minecraft Bedrock servers integration

### DISCUSSION THREAD

**Discussion by Alemiz112** [NamelessMC/Nameless#2153] [Ideas]
It would be nice to update Minecraft server queries to support mcbe servers.

**Comment by samerton** [comment]
Coming in the next pre-release :)

**Comment by Alemiz112** [comment]
Also this api can be used for external queries. 
https://api.mcsrvstat.us/

**Comment by samerton** [comment]
The Nameless external query has already been updated to add a new endpoint for Bedrock servers, just the Nameless side that needs updating to support it now.

**Comment by edgiru** [comment]
Hello
Tell me support is already added in the new version?
Minecraft Bedrock Edition & NukkitX

**Comment by Derkades** [comment]
Support for bedrock server status has beed added for pr13
